src/vs/server/node/remoteAgentEnvironmentImpl.ts

```diff
@@ -290,7 +292,7 @@ export class RemoteAgentEnvironmentChannel implements IServerChannel {
 			logsPath: URI.file(this._environmentService.logsPath),
 			extensionsPath: URI.file(this._environmentService.extensionsPath!),
 			extensionHostLogsPath: URI.file(join(this._environmentService.logsPath, `exthost${RemoteAgentEnvironmentChannel._namePool++}`)),
-			globalStorageHome: this._environmentService.globalStorageHome,
+			globalStorageHome: this._userDataProfilesService.defaultProfile.globalStorageHome,
```

Comment:
Why default profile and not the users profile? I think on the server, each window that connects needs to send the associated profile?

Reply:
I will consider while looking at profiles for remote

Reply:
I remember that we can skip profiles in remote because, we do not want extension storage (folder on disk) to be included in profile. And remote only has extension storage not extension state.